### ISSUE TITLE
API for Moving to Final Releases

### ISSUE THREAD

**Issue by belph** [k-bx/python-semver#65]
(Disclaimer: this is certainly something that could be "self-rolled" from what is already exposed by `python-semver`, but it seems like a useful API to provide)
To my knowledge, there is a gap in the `bump_*` family of functions. Consider the version `v = 1.2.3-rc.4+build.5`:
```
bump_major(v)      -> 2.0.0
bump_minor(v)      -> 1.3.0
bump_patch(v)      -> 1.2.4
bump_prerelease(v) -> 1.2.3-rc.5
bump_build(v)      -> 1.2.3-rc.4+build.6
```
What I think would be useful is something along the lines of a `finalize_version` function, which would convert both `1.2.3` and `1.2.3-rc.4+build.5` into `1.2.3` (i.e. the corresponding final version). Alternatively, if we want to look at [`node-semver`][node-semver] as inspiration, the behavior of `bump_patch` could be changed such that `1.2.3-rc.4` would bump to `1.2.3`, while `1.2.3` would bump to `1.2.4`. The intuition is that, since the [spec][spec] says that these final versions are greater than the prerelease builds, it would make sense to be able to "bump" to them.

Would you agree that this is worth adding?

[node-semver]: https://github.com/npm/node-semver/blob/ec6f97ad9bb4e8ffe9100c2f331f4e0015a24212/semver.js#L441-L444
[spec]: http://semver.org/spec/v2.0.0.html#spec-item-11

**Comment by k-bx** [comment]
Great idea! PR is welcome:)

**Comment by belph** [comment]
Which would make more sense in your opinion, the `finalize_version` function, or the change to `bump_patch`'s behavior?

**Comment by k-bx** [comment]
‘finalize_version’ makes more sense to me

**Comment by belph** [comment]
@k-bx: Alright, I've opened #66 to address this, and the Travis build has passed. Thanks for being receptive to the addition!

**Comment by k-bx** [comment]
@belph thank you! Released under 2.7.9 https://pypi.python.org/pypi/semver
